Integrations/integration-Zendesk.yml

```diff
@@ -394,19 +392,28 @@ script:
                     }
                 }
             };
-
         res = sendRequest('PUT', 'tickets/' + ticketId + '.json', JSON.stringify(body));
-
         md += 'Comment added successfully.\n';
+        return ( {'ContentsFormat': formats.json, 'Type': entryTypes.note, 'Contents': res, "HumanReadable": md} );
+    };
 
+    var addUser = function() {
+        var body = {
+                user: {
+                    name: args.name,
+                    email: args.email
+                }
+        };
+
+        res = sendRequest('POST', 'users/create_or_update.json', JSON.stringify(body));
+        md = 'Zendesk user added successfuly.\n';
```

Comment:
I think it is better to write the name/email of that user. will be clearier after.

Reply:
added